dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -127,10 +127,12 @@ DomainParticipantImpl::~DomainParticipantImpl()
     DDS::Security::SecurityException se;
     if (!access->return_permissions_handle(perm_handle_, se)) {
       if (DCPS::security_debug.auth_warn) {
-        ACE_ERROR((LM_ERROR,
-                   ACE_TEXT("(%P|%t) ERROR: DomainParticipantImpl::~DomainParticipantImpl: ")
-                   ACE_TEXT("Unable to return permissions handle. SecurityException[%d.%d]: %C\n"),
-                   se.code, se.minor_code, se.message.in()));
+        if (DCPS_debug_level > 0) {
```

Comment:
`DCPS::security_debug` are log settings that are separate from `DCPSDebugLevel` like the transport debug level is. Please revert this.

Reply:
Sorry, missed that, reverted